### PR TITLE
update the translation of `Object.is()`

### PR DESCRIPTION
**Pull request by robert-jx**
### Description
update the translation of `Object.is()`

### Motivation

update the translation of `Object.is()`
### Additional details

none

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/object/is/index.md [hunk 1]
```diff
@@ -5,95 +5,77 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/is
...
 - 都是数字且
+
   - 都是 `+0`
   - 都是 `-0`
   - 都是 {{jsxref("NaN")}}
-  - 都是同一个值，非零且都不是 {{jsxref("NaN")}}
+  - 都有相同的值，非零且都不是 {{jsxref("NaN")}}
 
-`Object.is()` 与 {{jsxref("Operators", "==", "#相等运算符")}} 不同。`==` 运算符在判断相等前对两边的变量（如果它们不是同一类型）进行强制转换（这种行为将 `"" == false` 判断为 `true`），而 `Object.is` 不会强制转换两边的值。
+`Object.is()` 不等同于 [`==`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Equality) 运算符。在测试相等性之前，`==` 运算符会对两边的变量（如果它们不是一个类型）应用各种强制转换（导致诸如 `"" == false` 的行为为 `true`），但是 `Object.is()` 不会强制转换任何一个值。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/is/index.md:47]:
```suggestion
`Object.is()` 不等同于 [`==`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Equality) 运算符。在测试相等性之前，`==` 运算符会对两边的变量（如果它们不是一个类型）应用各种强制转换（这会导致像 `"" == false` 为 `true` 的现象），但是 `Object.is()` 不会强制转换任何一个值。
```

Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/is/index.md:47]:
```suggestion
`Object.is()` 与 [`==`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Equality) 运算符并不等价。`==` 运算符在测试相等性之前，会对两个操作数进行类型转换（如果它们不是相同的类型），这可能会导致一些非预期的行为，例如 `"" == false` 的结果是 `true`，但是 `Object.is()` 不会对其操作数进行类型转换。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/is/index.md [hunk 2]
```diff
@@ -5,95 +5,77 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/is
...
 
-`Object.is()` 与 {{jsxref("Operators", "===", "#全等运算符")}} 也不相同。差别是它们对待有符号的零和 NaN 不同，例如，`===` 运算符（也包括 `==` 运算符）将数字 `-0` 和 `+0` 视为相等，而将 {{jsxref("Number.NaN")}} 与 {{jsxref("NaN")}} 视为不相等。
+`Object.is()` 也是*不*等价于 [`===`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Strict_equality) 运算符。`Object.is()` 和 `===` 之间的唯一区别在于它们处理有符号的 0 和 `NaN` 值的时候。`===` 运算符（和 `==` 运算符）将数值 `-0` 和 `+0` 视为相等，但是会将 {{jsxref("NaN")}} 视为不相等。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/is/index.md:49]:
```suggestion
`Object.is()` 也*不*等价于 [`===`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Strict_equality) 运算符。`Object.is()` 和 `===` 之间的唯一区别在于它们处理带符号的 0 和 `NaN` 值的时候。`===` 运算符（和 `==` 运算符）将数值 `-0` 和 `+0` 视为相等，但是会将 {{jsxref("NaN")}} 视为彼此不相等。
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions.

File: files/zh-cn/web/javascript/reference/global_objects/object/is/index.md [hunk 3]
```diff
@@ -5,95 +5,77 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/is
...
-- 都是 {{jsxref("null")}}
-- 都是 `true` 或都是 `false`
-- 都是相同长度、相同字符、按相同顺序排列的字符串
-- 都是相同对象（意味着都是同一个对象的值引用）
+- 都是 [`null`](/zh-CN/docs/Web/JavaScript/Reference/Operators/null)
+- 都是 `true` 或者都是 `false`
+- 都是长度相同、字符相同、顺序相同的字符串
+- 都是相同的对象（意味着两个值都引用了内存中的同一对象）
+- 都是 [BigInts](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/BigInt) 且具有相同的数值
+- 都是 [symbols](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol) 且引用相同的 symbol 值
```
Comment (fwqaaq) [inline files/zh-cn/web/javascript/reference/global_objects/object/is/index.md:39]:
```suggestion
- 都是 [BigInt](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/BigInt) 且具有相同的数值
- 都是 [symbol](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol) 且引用相同的 symbol 值
```

单数

File: files/zh-cn/web/javascript/reference/global_objects/object/is/index.md [hunk 4]
```diff
@@ -106,5 +88,5 @@ if (!Object.is) {
 
 ## 参见
 
-- [Polyfill of `Object.is` in `core-js`](https://github.com/zloirock/core-js#ecmascript-object)
-- [JavaScript 中的相等性判断](/zh-CN/docs/Web/JavaScript/Equality_comparisons_and_sameness)
+- [`core-js` 中 `Object.is` 的 Polyfill](https://github.com/zloirock/core-js#ecmascript-object)
+- [JavaScript 中的相等性判断](/zh-CN/docs/Web/JavaScript/Equality_comparisons_and_sameness)——三种内置相同功能的比较
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/is/index.md:92]:
```suggestion
- [JavaScript 中的相等性判断](/zh-CN/docs/Web/JavaScript/Equality_comparisons_and_sameness)——三种内置相等性工具的比较
```

**Review by yin1999 — APPROVED** [review]
Thanks!

**Pull request merged**